### ISSUE TITLE
Provide PyTorch module that cuts off predicted data < 0s

### ISSUE THREAD

**Issue by ar4** [ar4/deepwave#10]
The source wavelet often begins before time 0, the start of the recorded data in seismic surveys. Deepwave produces receiver amplitudes of the same length as the source amplitude, so in these cases the source amplitudes, and thus the predicted receiver amplitudes, will start before the true receiver amplitudes do. Comparing the predicted and true amplitudes will therefore result in an error unless the beginning of the predicted amplitudes (before time 0) is cut off. This could be done using a new PyTorch module that cuts the predicted amplitudes so that they are the same length as the true amplitudes. It could be used like this:
```python
loss = criterion(new_module(predicted, true))
```
(where `new_module` is the new module and I don't know what it should be called). This could probably be a pure Python module that allows autograd to calculate its affect on gradients.
